ledger-core/v2/virtual/integration/conveyor/bargin_call_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package conveyor
+
+import (
+	"testing"
+
+	"github.com/gojuno/minimock/v3"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine/smsync"
+	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/v2/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/v2/testutils/slotdebugger"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/injector"
+)
+
+type StepCallback func(ctx smachine.ExecutionContext)
+
+type SMWithCallback struct {
```

Comment:
Can you extract that SM to separate file and move it to utils folder?

Reply:
done